src/test/java/com/task10/crudapi_login/mapper/ClientMapperTest.java

```diff
@@ -53,4 +54,14 @@ class ClientMapperTest {
         Optional<Client> clients = clientMapper.findById(99);
         assertThat(clients).isEmpty();
     }
+
+    @Test
+    @DataSet(value = "datasets/clients.yml")
+    @ExpectedDataSet(value = "datasets/insert_clients.yml", ignoreCols = "id")
+    @Transactional
+    void 顧客データを新規登録すること() {
+        Client client = new Client("石田四郎", 45, "08044444444");
+        clientMapper.insert(client);
+        assertThat(client.getId()).isGreaterThan(3);
```

Comment:
こちらですが、既に`src/test/resources/datasets/insert_clients.yml` に同様のデータが存在しているかつidが3以上のデータが存在するようです。
その場合、insertが成功しているかいないかに関わらず常にテストは成功すると思います！
やるならば、テストデータとは別のデータのClientインスタンスを作成して、そのデータが正しく登録されていることを検証する必要があります！
以下は修正後のテストコードのイメージです。参考にしてください🙏

```suggestion
 Client client = new Client("石田四郎", 45, "08044444444");
    clientMapper.insert(client);

    // データベースから新規登録したデータを取得
    Client savedClient = clientMapper.findById(client.getId());

    // 期待される値が正しいか検証
    assertThat(savedClient).isNotNull();
    assertThat(savedClient.getName()).isEqualTo(client.getName());
    assertThat(savedClient.getAge()).isEqualTo(client.getAge());
    assertThat(savedClient.getPhoneNumber()).isEqualTo(client.getPhoneNumber());```

Reply:
ご指摘ありがとうございます！
修正いたしました🙇‍♀️
e01af4b86e385737f0657d28ffb1f60bf38fe758

### 動作確認
<img width="1652" alt="スクリーンショット 2023-12-22 14 49 09" src="https://github.com/minori-oya/task10/assets/138114043/ff1030a7-1223-43b2-87d2-041e057f5740">